source/screens/LoginScreen.tsx

```diff
@@ -373,6 +378,8 @@ function LoginScreen(props) {
             primary={false}
             showBackButton={false}
             colorSchema="red"
+            isSubstep={false}
+            onBack={undefined}
```

Comment:
wouldn't it be better to make `onBack` optional in the CloseModalButton component?

Reply:
Yep, se my other answer.